src/common/services/git.js

```diff
@@ -7,37 +7,28 @@ import execute from '../utils/exec'
 export const GIT_SWITCH_PATH = path.join(os.homedir(), '.git-switch')
 export const POST_COMMIT_BASE = '#!/bin/bash\n\n/bin/bash "$(dirname $0)"/post-commit.git-switch'
 
-const defaultAuthorAndCommitter = {
-  author: { name: '', email: '' },
-  committer: { name: '', email: '' }
-}
-
 export async function setAuthor(name, email) {
-  await execute(`git config --global author.name "${name}"`)
-  await execute(`git config --global author.email "${email}"`)
-}
-
-export async function setCommitter(name, email) {
   await execute(`git config --global user.name "${name}"`)
   await execute(`git config --global user.email "${email}"`)
 }
 
-export async function updateAuthorAndCommitter(users) {
-  const activeUsers = users.filter(u => u.active)
-  if (!activeUsers.length)
-    return defaultAuthorAndCommitter
+export async function setCoAuthors(coAuthors) {
```

Comment:
Nice call on just using one `setCoAuthors` each time instead of a separate set/update.